test/runtests.jl

```diff
@@ -159,25 +160,26 @@ module NullableTestEnum
 end
 
 if VERSION < v"0.7.0-DEV.2797"
-    # showcompact(io::IO, x::Nullable)
+    showc(io, x) = show(IOContext(io, :compact => true), x)
```

Comment:
This only applies to old versions: it should probably continue to use `showcompact`, right?

Reply:
True! Fixed.